src/constants/featureFlags.ts

```diff
@@ -1,3 +1,4 @@
 export const FEATURE_FLAGS = {
   STYLING_REVAMP: "styles",
+  HOMEPAGE_TEMPLATES: "homepage_new_templates",
```

Comment:
Just to confirm ya moving forward we use this FF? Ie any agency will either see all of the homepage changes or none of the homepage changes?

Reply:
yeap, cleared with @jandonyada for first pass